test/runtests.jl

```diff
@@ -43,3 +43,17 @@ end
     r1[] = 3
     @test r2[] == 4
 end
+
+@testset "throttle" begin
+    obs = Observable(1)
+    throttled = throttle(10, obs)
+    @test throttled[] == 1
+    obs[] = 2
+    sleep(0.1)
+    @test throttled[] == 2
+    obs[] = 3
+    sleep(5)
+    @test throttled[] == 2
+    sleep(10)
```

Comment:
I think these numbers are too big :sweat_smile: But then I'm not the most patient person.

Reply:
lol, I sped  it up by a factor of five.